backend/src/main/java/com/capstone/moneytree/controller/StockController.java

```diff
@@ -1,23 +1,83 @@
 package com.capstone.moneytree.controller;
 
+import com.capstone.moneytree.service.api.StockMarketDataService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 
-import com.capstone.moneytree.facade.StockMarketDataFacade;
 import com.capstone.moneytree.service.api.StockService;
+import pl.zankowski.iextrading4j.api.stocks.*;
+import pl.zankowski.iextrading4j.api.stocks.v1.BatchStocks;
+import pl.zankowski.iextrading4j.api.stocks.v1.KeyStats;
+import pl.zankowski.iextrading4j.api.stocks.v1.News;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+import java.util.List;
 
 @RestController
+@RequestMapping("/api/stockmarket")
 public class StockController extends ApiController {
 
-   private StockMarketDataFacade stockMarketDataFacade;
-   private StockService stockService;
+   @Autowired
+   private StockMarketDataService stockMarketDataService;
    private static final Logger LOG = LoggerFactory.getLogger(StockController.class);
 
-   @Autowired
-   public StockController(StockService stockService){
-      this.stockService = stockService;
+   /**
+    * Obtain json object containing company quote, info, news, keystats and logo
+    *
+    * @param symbol
+    * @return batchStocks
+    */
+   @GetMapping("/batch/{symbol}")
+   public ResponseEntity<BatchStocks> getBatchStocksBySymbol(@Valid @PathVariable(name = "symbol") String symbol) {
+      BatchStocks batchStocks = stockMarketDataService.getBatchStocksBySymbol(symbol);
```

Comment:
Is there a way to convert this BatchStock from IEXCloud library to the Stock class that we actually persist in NEO4J.

So front end can simply receive our Stock object. WDYT?

Reply:
Actually, now that I think about it. There is no need. Stock class is the stock bought by the user. BatchStock is what the front end needs with all info about stocks